src/mod/server.mod/server.h

```diff
@@ -131,6 +131,12 @@ enum {
   SASL_MECHANISM_NUM
 };
 
-extern char const *SASL_MECHANISMS[];
+/* Available sasl mechanisms. */
+char const *SASL_MECHANISMS[SASL_MECHANISM_NUM] = {
```

Comment:
designated initializers are C99, did we decide to go with C99?

Reply:
Indeed we did!